File: src/main/java/net/hectus/neobb/turn/person_game/block/PTBambooButton.java
package net.hectus.neobb.turn.person_game.block;

import net.hectus.neobb.player.NeoPlayer;
import net.hectus.neobb.turn.default_game.block.BlockTurn;
import net.hectus.neobb.turn.person_game.categorization.CounterCategory;
import org.bukkit.block.Block;

public class PTBambooButton extends BlockTurn implements CounterCategory {
    public PTBambooButton(NeoPlayer player) { super(player); }
    public PTBambooButton(Block data, NeoPlayer player) { super(data, player); }

    @Override
    public double damage() {
        return 2;
    }
}

File: src/main/kotlin/net/hectus/neobb/modes/turn/person_game/block/PTBambooButton.kt
package net.hectus.neobb.modes.turn.person_game.block

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.modes.turn.default_game.block.BlockTurn
import net.hectus.neobb.modes.turn.person_game.categorization.CounterCategory
import net.hectus.neobb.player.NeoPlayer
import org.bukkit.block.Block

class PTBambooButton(data: Block?, cord: Cord?, player: NeoPlayer?) : BlockTurn(data, cord, player), CounterCategory {
    override val damage: Double = 2.0
}
